src/components/dialogs/advanced-search.vue

```diff
@@ -0,0 +1,263 @@
+            </v-card-title>
+
+            <v-card-text class="copy-normal pt-5">
+              To view the details and status of this Name Request, please enter the <strong>email address or phone
+              number</strong> of the Applicant.
+            </v-card-text>
+
+            <AdvancedSearchRetrieve
+              class="mt-6"
+              :promptSubmit="toggleRetrieveSubmit"
+              @closeDialog="emitClose()"
+            />
+          </v-tab-item>
+
+          <!-- Invalid Results -->
+          <v-tab-item>
+            <v-card-text class="copy-normal results-text">
+              <h3>Too Many Name Requests Found</h3>
+              <span class="my-6">Your search returned more than 1000 results.</span>
+              <span class="mb-4">Please return to the previous screen and narrow your search by selecting more than
+                  one search criteria and/or reducing your date range.</span>
+            </v-card-text>
+          </v-tab-item>
+
+        </v-tabs-items>
+      </v-tabs>
+
+      <v-card-actions id="advanced-search-actions" class="justify-center pa-0">
+          <v-btn
+            id="adv-search-btn-1"
+            class="mr-2"
+            :class="{ 'button-blue': isTabResultsTable || isTabRetrieve }"
+            text
+            outlined
+            @click="advSearchBtn1Actions()"
+          >
+            <span class="px-1">
+              <v-icon v-if="!isTabSearchForm">mdi-chevron-left</v-icon>
+              {{ advSearchBtn1 }}
+              <v-icon v-if="isTabSearchForm">mdi-chevron-right</v-icon>
+            </span>
+          </v-btn>
+          <v-btn
+            id="adv-search-btn-2"
+            :class="{ 'button-blue' : !isTabRetrieve }"
+            text
+            outlined
+            @click="advSearchBtn2Actions()"
+          >
+            <span>
+              {{ cancelBtnText }}<v-icon v-if="isTabRetrieve">mdi-chevron-right</v-icon>
+            </span>
+          </v-btn>
+      </v-card-actions>
+    </v-card>
+
+  </v-dialog>
+</template>
+
+<script lang='ts'>
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { sleep } from '@/plugins'
+import { AdvancedSearchForm, AdvancedSearchRetrieve, AdvancedSearchTable } from '@/components/advanced-search'
+
+// Interfaces & Enums
+import { AdvancedSearchI } from '@/interfaces/models'
+import { AdvancedSearchTabs } from '@/enums'
+
+@Component({
+  components: {
+    AdvancedSearchForm,
+    AdvancedSearchRetrieve,
+    AdvancedSearchTable
+  }
+})
+export default class AdvancedSearch extends Vue {
+  /** Prop to display the dialog. */
+  @Prop() readonly dialog: boolean
+
+  /** Prop to provide attachment selector. */
+  @Prop() readonly attach: string
+
+  // Local Properties
+  private advSearchTab = 0
+  private toggleSearchSubmit = false
+  private toggleRetrieveSubmit = false
+  private invalidSearch = false
+  private readonly AdvancedSearchTabs = AdvancedSearchTabs
```

Comment:
I've been sticking these at the top of the class with comment `// Declarations for template`, if you want to copy that. (Also just readonly is fine.)

Reply:
For sure, i'm good with that!

Reply:
fixed!